src/validation.js

```diff
@@ -121,13 +125,16 @@ export default class AuroFormValidation {
     this.getInputElements(elem);
     this.getAuroInputs(elem);
 
+    const noInputElement = !this.auroInputElements && !this.inputElements;
+
     // Validate only if noValidate is not true and the input does not have focus
+    const noInputValidation = !elem.contains(document.activeElement) && elem.value === undefined && noInputElement;
```

Comment:
Per our conversation around how auro-select validation is handled via the value being stored as a string, please revisit the changed logic here.